src/components/shared/Tabs.jsx

```diff
@@ -0,0 +1,59 @@
+import classnames from 'classnames'
+import React, { useState } from 'react'
+
+import classNames from './Tabs.scss'
+
+export const Tabs = ({ children, initialTab = 0, ...props }) => {
+  const [tab, setTab] = useState(children[initialTab])
+
+  return (
+    <section className={classNames.root}>
+      <TabSelector
+        setTab={setTab}
+        tab={tab}
+        tabs={children}
+      />
+      <TabContent tab={tab} />
+    </section>
+  )
+}
+
+const TabSelector = ({ setTab, tab, tabs, ...props }) => {
```

Comment:
rename `tab` to `selectedTab`? `setTab` to `onTabSelected` or `onClick`? `tabs` to `children`?